2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_setup.py:_flush():67] Configure stats pid to 2940751
2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_setup.py:_flush():67] Loading settings from /home/guswls/.config/wandb/settings
2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_setup.py:_flush():67] Loading settings from /home/guswls/EEN4015_Intro2ML/pbl-2/wandb/settings
2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /home/guswls/EEN4015_Intro2ML/pbl-2/wandb/run-20250411_132600-jslye7bc/logs/debug.log
2025-04-11 13:26:00,468 INFO    MainThread:2940751 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /home/guswls/EEN4015_Intro2ML/pbl-2/wandb/run-20250411_132600-jslye7bc/logs/debug-internal.log
2025-04-11 13:26:00,469 INFO    MainThread:2940751 [wandb_init.py:monkeypatch_ipython():581] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f162fa55410>
2025-04-11 13:26:00,469 INFO    MainThread:2940751 [wandb_init.py:init():781] calling init triggers
2025-04-11 13:26:00,469 INFO    MainThread:2940751 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-04-11 13:26:00,469 INFO    MainThread:2940751 [wandb_init.py:init():809] starting backend
2025-04-11 13:26:00,469 INFO    MainThread:2940751 [wandb_init.py:init():813] sending inform_init request
2025-04-11 13:26:00,472 INFO    MainThread:2940751 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-04-11 13:26:00,472 INFO    MainThread:2940751 [wandb_init.py:init():823] backend started and connected
2025-04-11 13:26:00,478 INFO    MainThread:2940751 [wandb_run.py:_label_probe_notebook():1267] probe notebook
2025-04-11 13:26:00,478 INFO    MainThread:2940751 [wandb_run.py:_label_probe_notebook():1277] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-04-11 13:26:00,478 INFO    MainThread:2940751 [wandb_init.py:init():915] updated telemetry
2025-04-11 13:26:00,485 INFO    MainThread:2940751 [wandb_init.py:init():939] communicating run to backend with 90.0 second timeout
2025-04-11 13:26:01,199 INFO    MainThread:2940751 [wandb_init.py:init():1014] starting run threads in backend
2025-04-11 13:26:01,423 INFO    MainThread:2940751 [wandb_run.py:_console_start():2454] atexit reg
2025-04-11 13:26:01,423 INFO    MainThread:2940751 [wandb_run.py:_redirect():2306] redirect: wrap_raw
2025-04-11 13:26:01,424 INFO    MainThread:2940751 [wandb_run.py:_redirect():2371] Wrapping output streams.
2025-04-11 13:26:01,424 INFO    MainThread:2940751 [wandb_run.py:_redirect():2394] Redirects installed.
2025-04-11 13:26:01,426 INFO    MainThread:2940751 [wandb_init.py:init():1056] run started, returning control to user process
2025-04-11 13:26:01,428 INFO    MainThread:2940751 [wandb_run.py:_config_callback():1327] config_cb None None {'model': 'resnet18', 'batch_size': 128, 'num_epochs': 20, 'learning_rate': 0.001, 'optimizer': 'Adam', 'seed': 2025, 'deterministic': False}
2025-04-11 13:26:04,081 INFO    MainThread:2940751 [wandb_watch.py:_watch():71] Watching
2025-04-11 13:27:21,752 INFO    MainThread:2940751 [jupyter.py:save_ipynb():384] [no run ID] not saving jupyter notebook
2025-04-11 13:27:21,753 INFO    MainThread:2940751 [wandb_init.py:_pause_backend():546] [no run ID] pausing backend
2025-04-11 13:27:23,255 INFO    MainThread:2940751 [wandb_init.py:_resume_backend():551] [no run ID] resuming backend
2025-04-11 13:27:23,271 INFO    MainThread:2940751 [wandb_run.py:_finish():2189] finishing run sokjh1310-hanyang-university/PBL-2/jslye7bc
2025-04-11 13:27:23,272 INFO    MainThread:2940751 [jupyter.py:save_history():460] not saving jupyter history
2025-04-11 13:27:23,272 INFO    MainThread:2940751 [jupyter.py:save_ipynb():384] not saving jupyter notebook
2025-04-11 13:27:23,272 INFO    MainThread:2940751 [wandb_init.py:_jupyter_teardown():563] cleaning up jupyter logic
2025-04-11 13:27:23,272 INFO    MainThread:2940751 [wandb_run.py:_atexit_cleanup():2419] got exitcode: 0
2025-04-11 13:27:23,272 INFO    MainThread:2940751 [wandb_run.py:_restore():2401] restore
2025-04-11 13:27:23,272 INFO    MainThread:2940751 [wandb_run.py:_restore():2407] restore done
2025-04-11 13:27:24,734 INFO    MainThread:2940751 [wandb_run.py:_footer_history_summary_info():4064] rendering history
2025-04-11 13:27:24,734 INFO    MainThread:2940751 [wandb_run.py:_footer_history_summary_info():4096] rendering summary
2025-04-11 13:27:24,735 INFO    MainThread:2940751 [wandb_run.py:_footer_sync_info():4025] logging synced files
